src/Shapefile.jl

```diff
@@ -322,7 +319,7 @@ function Base.read(io::IO, ::Type{Handle})
     mmin = read(io, Float64)
     mmax = read(io, Float64)
     jltype = SHAPETYPE[shapeType]
-    shapes = Vector{Union{jltype,Missing}}(undef, 0)
```

Comment:
If anyone wants to try this:

I temporarily changed this `shapes = Any[]`. With this change I can use the package to read all kinds of maps. After this was reverted in the last commit by @Sov-trotter this branch is not usable for me.

@Sov-trotter, unless you have a good reason, it would be nice if you could remove your commit, so that I can use this instead of my own.

Reply:
Oh. The git side on my local side is also a bit messy. I think a cleaner option here would be, branching from here? And make the changes accordingly in a new pr?

Reply:
It's fine now, thanks!